src/NServiceBus.AcceptanceTests/Sagas/When_a_base_class_message_handled_by_a_saga.cs

```diff
@@ -0,0 +1,94 @@
+﻿namespace NServiceBus.AcceptanceTests.Sagas
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_a_base_class_message_handled_by_a_saga : NServiceBusAcceptanceTest
```

Comment:
I'm not sure I understand what this test verifies and what part didn't work before this PR? It looks like the mapping and the sent message are both using `SecondSagaMessage` which is a subclass. Didn't that already work before?